cdap-docs/_common/common-build.sh

```diff
@@ -175,7 +181,10 @@ function build_license_pdfs() {
   version
   cd $SCRIPT_PATH
   PROJECT_VERSION_TRIMMED=${PROJECT_VERSION%%-SNAPSHOT*}
-  rm -rf $SCRIPT_PATH/$LICENSES_PDF
+  if [ -d "${SCRIPT_PATH}/${LICENSES_PDF}" ]
+  then
+    rm -rf $SCRIPT_PATH/$LICENSES_PDF
+  fi
```

Comment:
Why replace one line with four? Especially that have two with a path that needs to be the same in two places.